src/components/client/ClientsTable.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import ActionsTable from '../global/ActionsTable';
+
+export default class ClientsTable extends React.Component {
+  static defaultProps = {
+    url: 'https://lesewert.herokuapp.com/api/v1',
```

Comment:
why are you using defaultprops and not just a const?

Reply:
Because this link is used more than 1 time in this file, and in the future, if you need to change eg version of api you have to change it only in one place. But now we have this in ProjectsTable and ClientsTable, I think we need to consider creation one config file. Will create ticket for it.